tests/integration/components/lt-body-test.js

```diff
@@ -148,16 +148,20 @@ test('row expansion - multiple', function(assert) {
 });
 
 test('row actions', function(assert) {
-  assert.expect(2);
+  assert.expect(4);
 
   this.set('table', new Table(Columns, createUsers(1)));
   this.on('onRowClick', (row) => assert.ok(row));
   this.on('onRowDoubleClick', (row) => assert.ok(row));
-  this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions onRowClick=(action 'onRowClick') onRowDoubleClick=(action 'onRowDoubleClick')}}`);
+  this.on('onRowMouseEnter', (row) => assert.ok(row));
+  this.on('onRowMouseLeave', (row) => assert.ok(row));
+  this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions onRowClick=(action 'onRowClick') onRowDoubleClick=(action 'onRowDoubleClick') onRowMouseEnter=(action 'onRowMouseEnter') onRowMouseLeave=(action 'onRowMouseLeave')}}`);
 
   let row = find('tr');
   click(row);
   triggerEvent(row, 'dblclick');
+  triggerEvent(row, 'mouseenter');
+  triggerEvent(row, 'mouseleave');
```

Comment:
And then lastly use `await` for all interaction helpers here as well. This will make sure that the test runner waits until the world has settled and all expected assertions are actually run.